infrastructure/repository/authzed/SpiceDbAccessRepository.go

```diff
@@ -96,7 +96,7 @@ func (s *SpiceDbAccessRepository) ModifySeats(assignedSubjectIDs []domain.Subjec
 
 	// Step 2 Add license changes
 	relationshipUpdates, preconditions = addLicenseVersionSwap(relationshipUpdates, preconditions, license, assignedCount)
-
+	glog.Infof("Trying to assign %s and unassign %s seats on license %s for org %s with %d of %d seats currently in use.", assignedSubjectIDs, removedSubjectIDs, license.ServiceID, license.OrgID, license.InUse, license.MaxSeats)
```

Comment:
didn't want to add too many if/elses to check if the slices are empty, so added a combined log message.